src/commands/detectorDiagnostics.ts

```diff
@@ -0,0 +1,144 @@
+        async () => {
+          const clusterAppLensData = await getAppLensDetectorData(cloudTarget);
+
+          if (clusterAppLensData) {
+            await createDetectorWebView(clsutername, clusterAppLensData, extensionPath);
+          }
+        }
+    );
+}
+
+async function longRunning<T>(title: string, action: () => Promise<T>): Promise<T> {
+  const options = {
+      location: vscode.ProgressLocation.Notification,
+      title: title
+  };
+  return await vscode.window.withProgress(options, (_) => action());
+}
+
+async function createDetectorWebView(
+  clusterName: string,
+  clusterAppLensData: AppLensARMresponse,
+  extensionPath: string) {
+    const panel = vscode.window.createWebviewPanel(
+      'AKS Diagnostics',
+      'AKS diagnostics view for: ' + clusterName,
+      vscode.ViewColumn.Active,
+      {
+        enableScripts: true,
+        enableCommandUris: true
+      }
+    );
+
+    panel.webview.html = getWebviewContent(clusterAppLensData, extensionPath);
+}
+
+async function getAppLensDetectorData(
+  clusterTarget: k8s.CloudExplorerV1.CloudExplorerResourceNode
+  ): Promise<AppLensARMresponse | undefined> {
+  const apiResult = await getNetworkConnectivityInfo(clusterTarget);
+
+  if (apiResult.succeeded) {
+    return apiResult.result;
+  } else if (!apiResult.succeeded) {
+    vscode.window.showInformationMessage(apiResult.error);
+  }
+  return;
+}
+
+async function getNetworkConnectivityInfo(
+  target: k8s.CloudExplorerV1.CloudExplorerResourceNode
+  ): Promise<Errorable<AppLensARMresponse>> {
+  try {
+      const client = new ResourceManagementClient(target.cloudResource.root.credentials, target.cloudResource.root.subscriptionId);
+      const resourceGroup = target.cloudResource.armId.split("/")[4];
+      const networkConnectivityInfo = await client.resources.get(
+        resourceGroup, target.cloudResource.resource.type,
+        target.cloudResource.resource.name, "detectors", "networkconnectivity", "2019-08-01");
+
+      return { succeeded: true, result: <AppLensARMresponse> networkConnectivityInfo};
+    } catch (ex) {
+      return { succeeded: false, error:  `Error invoking network connectivity detector: ${ex}` };
+    }
+}
+
+function getWebviewContent(
+  clusterdata: AppLensARMresponse,
+  vscodeExtensionPath: string) {
+    const webviewClusterData = clusterdata?.properties;
+
+    const stylePathOnDisk = vscode.Uri.file(path.join(vscodeExtensionPath, 'src', 'commands', 'style', "networkconnectivity", 'networkConnectivity.css'));
+    const htmlPathOnDisk = vscode.Uri.file(path.join(vscodeExtensionPath, 'src', 'commands', 'style', "networkconnectivity", 'networkConnectivity.html'));
+    const styleUri = stylePathOnDisk.with({ scheme: 'vscode-resource' });
+    const pathUri = htmlPathOnDisk.with({scheme: 'vscode-resource'});
+    const portalUrl = `https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/${clusterdata.id.split('detectors')[0]}aksDiagnostics`;
```

Comment:
💡 found the answer to this from one of the portal feedback links: so its as simple as this --  just a sample but this will does the trick `https://portal.azure.com/#resource/subscriptions/sub_id/resourceGroups/res_grp/overview`